Mathlib/Topology/Maps.lean

```diff
@@ -66,6 +66,10 @@ protected theorem Inducing.comp (hg : Inducing g) (hf : Inducing f) :
   ⟨by rw [hf.induced, hg.induced, induced_compose]⟩
 #align inducing.comp Inducing.comp
 
+theorem Inducing.of_comp_iff (hg : Inducing g) : Inducing (g ∘ f) ↔ Inducing f := by
```

Comment:
Isn't this `Inducing.inducing_iff` ?

Reply:
Also I think I like this terminology better, since all the lemmas about transfer of properties are named `Inducing.foo_iff`. Or maybe `Inducing.inducing_comp_iff` if you want `comp` to appear.

Reply:
Gah!!! I couldn't find that. It's definitely in the wrong place in the file, and it doesn't have `comp` in the name.

And I still can't find any of the others in this PR. 

As for naming, the library I think disagrees with you: [Function.Injective.of_comp_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Logic/Function/Basic.html#Function.Injective.of_comp_iff) and similarly for bijective, surjective, and OpenEmbedding.

Reply:
Yes I can't find the other ones either. And I'll concede the naming, even though it sounds a bit weird to me (why is there a `of` ?)

Reply:
I'd be okay with `comp_iff`, but I just tried to match the existing stuff.

Reply:
I've removed `Inducing.inducing_iff` in favor of the `.of_comp_iff` for uniformity across the library. If we want to bikeshed on Zulip to changes all these names, that's fine, but let's do it in a different PR.